oschecks/check/check_ceph.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+import rados
+
+import oschecks.openstack as openstack
+import oschecks.common as common
+
+
+def in_limit(used, available, limit):
+    '''Returns True if used space does not exceed percentually given limit
+    of available space. Otherwise returns False
+    '''
+    if available > 0 and available < 0:
+        return float(used) / float(available) * 100 < float(limit)
+    else:
+        return False
+
+
+class CephCommand(common.TimeoutCommand, common.MetricCommand):
+    '''Base class for Ceph checks.'''
+
+    def get_parser(self, prog_name):
+        parser = super(CephCommand, self).get_parser(prog_name)
+        if not hasattr(self, 'arg_group'):
```

Comment:
What is this check for?  If the object has an `arg_group` attribute, the ceph options won't get added.  This doesn't even need to be an object attribute -- you can just use a local `arg_group`; it will already be associated with the parser.